.vsts.release.yml

```diff
@@ -181,6 +181,35 @@ stages:
         artifactName: agent
         artifactType: container
 
+  ################################################################################
+  - job: build_linux_agent_arm64
+  ################################################################################
+    displayName: Linux Agent (ARM64)
+    condition: false
```

Comment:
Why's this disabled? If the idea is that we want to only build in CI for a while, we should hold off on included the ARM64 links in the release notes.

Reply:
My plan was to complete this PR, which will trigger attaching an artifact containing a "good" build of the agent. Then I'll grab that package, install it on a few machines, and immediately post a second PR enabling this job.

(The ARM 32-bit agents are the long pole on builds. If I can get a healthy build out of the CI pipeline, I will also be able to use that on the newer hardware to make future iteration faster.)